src/electron/views/screenshot/highlight-box.tsx

```diff
@@ -8,7 +8,7 @@ import { CSSProperties } from 'react';
 
 export interface HighlightBoxDeps {}
 export interface HighlightBoxProps {
-    deps: HighlightBoxDeps;
+    deps?: HighlightBoxDeps;
```

Comment:
We don't usually require that components be able to operate without their deps; if this component doesn't actually have any deps, it's best to just omit the deps prop and type entirely.

Reply:
Removed HighlightBoxDeps